docs/pages/preview/custom-build-config-schema.mdx

```diff
@@ -463,18 +466,92 @@ functions:
     # @info #
     shell: /bin/sh
     # @end #
+    command: echo "Hi!"
 ```
 
+### `functions.[function_name].supported_platforms`
+
+Used to define the supported platforms for a function. Defaults to all platforms. Allowed platforms: `darwin`, `linux`.
+
+For example, the function's supported platform is `darwin` (macOS):
+
+```yaml
+functions:
+  greetings:
+    name: Say Hi!
+    # @info #
+    supported_platforms: [darwin]
+    # @end #
+    command: echo "Hi!"
+```
+
+## `import`
+
+A config file path list used to import functions from other config files. Imported files cannot have the `build` section.
+
+For example, the following workflow imports two files and calls two imported functions - `say_hi` and `say_bye`.
+
+```yaml workflow.yml
+import:
+  - common-functions.yml
+  - another-file.yml
+
+build:
+  steps:
+    - say_hi
+    - say_bye
+```
+
+```yaml common-functions.yml
+functions:
+  say_hi:
+    name: Say Hi!
+    command: echo "Hi!"
+```
+
+```yaml another-file.yml
+functions:
+  say_bye:
+    name: Say bye :(
+    command: echo "Bye!"
+```
+
+## Functions
+
 ### Built-in EAS functions
 
-EAS provides a built-in reusable function called `upload_artifact` that you can use in a workflow without defining the function definition.
+EAS provides a set of built-in reusable functions that you can use in a workflow without defining the function definition.
+
+> **info** **Tip:** Any function that is built-in and provided by EAS must start with the prefix `eas/`.
+
+#### `eas/checkout`
+
+Checks out you project source files.
+
+For example, a workflow with the following `steps` will check out the project and list the files in the `assets` directory:
```

Comment:
```suggestion
For example, a workflow with the following `steps` will check out the project and list the files in the **assets** directory:
```

file/directory names to be bold for consistency as per our writing styelguide: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text